CHANGELOG.rst

```diff
@@ -17,6 +17,21 @@ This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 .. towncrier release notes start
 
+[1.10.7] - 2020-07-07
+^^^^^^^^^^^^^^^^^^^^^
+
+Features
```

Comment:
A feature in a patch release? 😬

Reply:
just a small addition to avoid further problems with the whitespace tokenizer...

Reply:
It's kind of in between for me but ok 👍  
https://tenor.com/6Drq.gif 😆